warehouse/manage/views.py

```diff
@@ -991,7 +1047,24 @@ def __init__(self, request):
 
     @property
     def default_response(self):
+        all_user_organizations = user_organizations(self.request)
         return {
+            "organizations": self.organization_service.get_organizations_by_user(
+                self.request.user.id
+            ),
+            **user_organizations(self.request),
```

Comment:
I assume the query would be cached, but we don't even need to make the 3 calls to SQLAlchemy in the function if we

```suggestion
            **all_user_organizations,
```